src/pages/hez-circ-supply.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
shouldn't the file be .jsx?

Reply:
Well it does not hurt that we have all pages with `.jsx` extension. I changed it.